src/VerifiableClaim.test.ts

```diff
@@ -0,0 +1,241 @@
+      name: ravenClaim.name,
+      author: ravenClaim.author,
+    })
+
+    const work2: BaseVerifiableClaim = makeClaim({
+      author: ravenClaim.author,
+      name: ravenClaim.name,
+    })
+
+    const id1 = await generateClaimId(work1).catch(getErrorMessage)
+    const id2 = await generateClaimId(work2).catch(getErrorMessage)
+
+    assert({
+      given: 'two claims with disordered keys',
+      should: 'have the same claims id',
+      actual: id1 === id2,
+      expected: true,
+    })
+  }
+
+  {
+    const work1: BaseVerifiableClaim = makeClaim({
+      name: ravenClaim.name,
+      author: ravenClaim.author,
+    })
+
+    const work2: BaseVerifiableClaim = makeClaim({
+      author: ravenClaim.author,
+      nAME: ravenClaim.name,
+    })
+
+    const id1 = await generateClaimId(work1)
+    const id2 = await generateClaimId(work2)
+
+    assert({
+      given: 'two claims with different keys casing',
+      should: 'NOT have the same claims id',
+      actual: id1 !== id2,
+      expected: true,
+    })
+  }
+
+  {
+    const workClaim = makeClaim({
+      name: ravenClaim.name,
+      author: ravenClaim.author,
+    })
+
+    const TheRavenId = '9bd16701a14884f129625f399ddad0b956b7883a5ef30142d0cda28f38988eca'
+
+    assert({
+      given: 'A work claim',
+      should: 'generate an id for the claim',
+      actual: await generateClaimId(workClaim),
+      expected: TheRavenId,
+    })
+
+    const identityClaim = makeClaim({ ...MyIdentity.claim })
+
+    assert({
+      given: 'an verifiable identity claim',
+      should: 'generate an id for the claim',
+      actual: await generateClaimId(identityClaim),
+      expected: 'b609122e7988bf3e9a4fd7b4336748bce1ae7134a7317e7009661c6936d55722',
+    })
+
+    const workClaim2 = makeClaim({
+      name: ravenBookClaim.name,
+      author: ravenBookClaim.author,
+      isbn: ravenBookClaim.isbn,
+      edition: ravenBookClaim.edition,
+    })
+
+    assert({
+      given: 'an extended work claim WITHOUT proper context',
+      should: 'generate the id without the extra attributes',
+      actual: await generateClaimId(workClaim2),
+      expected: TheRavenId,
+    })
+
+    assert({
+      given: 'an extended work claim WITH proper context',
+      should: 'generate a different id',
+      actual: (await generateClaimId({ ...workClaim2, '@context': externalContext })) !== TheRavenId,
+      expected: true,
+    })
+  }
+})
+
+describe('Claim.configureCreateVerifiableClaim with Ed25519 Issuer', async (assert: any) => {
+  const issuer = createIssuerFromPrivateKey(ed25519Base58PrivateKey)
+
+  {
+    const createWorkClaim = configureCreateVerifiableClaim({ issuer })
+    const verifiableClaim = await createWorkClaim(TheRavenClaim)
+
+    assert({
+      given: 'a claim',
+      should: 'create a Verifiable Claim',
+      actual: isVerifiableClaim(verifiableClaim),
+      expected: true,
+    })
+
+    assert({
+      given: 'a verififable work claim without any extended context',
```

Comment:
How about this, i've adjust the given/should and also simplified the actual/expected.

```
assert({
      given: 'a work claim, default work type and context,
      should: 'return a verifiable work claim with the correct context',
      actual: verifiableClaim['@context'],
      expected: { ...DefaultClaimContext, ...DefaultWorkClaimContext },
    })
```

I think we should also use `isWork` in a separate assert to verify that the type is correct.

If you agree with the above please update the other tests you added as well.